hummingbot/connector/connector_base.pyx

```diff
@@ -17,6 +15,14 @@ from hummingbot.core.network_iterator import NetworkIterator
 from hummingbot.core.rate_oracle.rate_oracle import RateOracle
 from hummingbot.core.utils.estimate_fee import estimate_fee
 
+if TYPE_CHECKING:
+    from hummingbot.client.config.client_config_map import ClientConfigMap
+    from hummingbot.client.config.config_helpers import ClientConfigAdapter
+
+NaN = float("nan")
+s_decimal_NaN = Decimal("nan")
+s_decimal_0 = Decimal(0)
```

Comment:
Why do we need this if we also have the following import line (line 8)?
`from hummingbot.connector.constants import NaN, s_decimal_NaN, s_decimal_0`

Reply:
Removed